packages/node_modules/@node-red/runtime/lib/api/flows.js

```diff
@@ -255,5 +259,83 @@ var api = module.exports = {
             }
         }
         return sendCredentials;
-    }
+    },
+    /**
+    * Gets running state of runtime flows
+    * @param {Object} opts
+    * @param {User} opts.user - the user calling the api
+    * @param {Object} opts.req - the request to log (optional)
+    * @return {{state:string, started:boolean}} - the current run state of the flows
+    * @memberof @node-red/runtime_flows
+    */
+    getState: async function(opts) {
+        runtime.log.audit({event: "flows.getState"}, opts.req);
+        const result = {
+            state: runtime.flows.started ? "started" : "stopped",
+            started: !!runtime.flows.started,
+            rev: runtime.flows.getFlows().rev
+        }
+        return result;
+    },
+    /**
+    * Sets running state of runtime flows
+    * @param {Object} opts
+    * @param {Object} opts.req - the request to log (optional)
+    * @param {User} opts.user - the user calling the api
+    * @param {string} opts.requestedState - the requested state. Valid values are "start" and "stop".
+    * @return {Promise<Flow>} - the active flow configuration
+    * @memberof @node-red/runtime_flows
+    */
+    setState: async function(opts) {
+        opts = opts || {};
+        const makeError = (error, errcode, statusCode) => {
+            const message = typeof error == "object" ? error.message : error
+            const err = typeof error == "object" ? error : new Error(message||"Unexpected Error")
+            err.status = err.status || statusCode || 400;
+            err.code = err.code || errcode || "unexpected_error"
+            runtime.log.audit({
+                event: "flows.setState",
+                state: opts.requestedState || "",
+                error: errcode || "unexpected_error",
+                message: err.code
+            }, opts.req);
+            return err
+        }
+
+        const getState = () => {
+            return {
+                state: runtime.flows.started ? "started" : "stopped",
+                started: !!runtime.flows.started,
+                rev: runtime.flows.getFlows().rev,
+            }
+        }
+
+        if(!runtime.settings.runtimeState || runtime.settings.runtimeState.enabled !== true) {
+            throw (makeError("Method Not Allowed", "not_allowed", 405))
+        }
+        switch (opts.requestedState) {
+            case "start":
+                try {
+                    try {
+                        runtime.settings.set('flowsRunStateRequested', opts.requestedState);
```

Comment:
```suggestion
                        runtime.settings.set('runtimeState', opts.requestedState);
```

I suggest calling the setting `runtimeState` - its the state of the runtime, not just a requested state.

I think I've suggested edits for all occurrences - but there could be ones I've missed.